activemodel/lib/active_model/errors.rb

```diff
@@ -147,9 +147,9 @@ def set(key, value)
 
     # Delete messages for +key+. Returns the deleted messages.
     #
-    #   person.errors[:name]    # => ["cannot be nil"]
+    #   person.errors[:name]        # => ["cannot be nil"]
```

Comment:
This is unrelated to the change you're actually making, and is just cosmetic git churn. If you want to token align the docs you're adding, fine, but please don't change unrelated docs.